src/views/components/landing-top.html

```diff
@@ -10,8 +10,8 @@
 	<div class="content">
 		<div class="container">
 			<section class="heading">
-				<h1>jsDelivr – Open Source CDN</h1>
-				<h2>free, fast, and reliable</h2>
+				<h1>A free CDN for Open Source Projects</h1>
+				<h2>Fast, global, reliable and automated</h2>
```

Comment:
I also don't like this, both are too long. Maybe something like this

![image](https://user-images.githubusercontent.com/6192491/82157278-c2d20100-9880-11ea-9086-6dd9b38ca7e3.png)

although I still like the current one better.

Reply:
Ok lets use your shorter version